src/openms/source/DATASTRUCTURES/LPWrapper.cpp

```diff
@@ -469,8 +469,12 @@ namespace OpenMS
     return solver_;
   }
 
+#if COINOR_SOLVER == 1
```

Comment:
to get rid of a compile warning? wouldn't add another compile path here

Reply:
I mean the function is macroed already. Why not the declaration?